projects/collection-editor-library/src/lib/services/editor/editor.service.ts

```diff
@@ -260,5 +263,61 @@ export class EditorService {
     this.telemetryService.error(telemetryErrorData);
   }
 
+  getAssetMedia(req?: object) {
```

Comment:
@rajnish-dargan We have already created this `getAssetMedia` method in the `question.service.ts` file. 
Please check the same  and use it from here : `question.service.ts`